nipype/interfaces/niftyreg/base.py

```diff
@@ -91,7 +99,7 @@ def _environ_update(self):
             self.inputs.omp_core_val = Undefined
 
     def check_version(self):
-        _version = self.version_from_command()
+        _version = self.version
```

Comment:
L106 assumes this is a `bytes` object. I think it's going to be a `str`?

Reply:
I think so. I got an error because it was a `str` so I removed the old L60: https://github.com/nipy/nipype/pull/3194/files/199dc2e3e1a6c5dd0f8f35b6a6497015325ee85f#diff-84c9861899b85298fccffbb72fe8fe9eL60.

I'll remove this one too.